odoo-shippable/scripts/build-image.sh

```diff
@@ -185,9 +185,10 @@ let g:signify_disable_by_default = 1
 EOF
 
 # Install and configure YouCompleteMe
+# Disable youcompleteme for default
 VIM_YOUCOMPLETEME_PATH="${HOME}/.vim/bundle/YouCompleteMe"
 git_clone_copy "${VIM_YOUCOMPLETEME_REPO}" "master" "." "${VIM_YOUCOMPLETEME_PATH}"
-(cd "${VIM_YOUCOMPLETEME_PATH}" && ./install.py)
+#(cd "${VIM_YOUCOMPLETEME_PATH}" && ./install.py)
```

Comment:
Yes, we can!
You can see it in the following image:

 - ![screen shot 2017-06-21 at 19 16 19](https://user-images.githubusercontent.com/6644187/27411614-592a4476-56b4-11e7-80cd-681d539ffd2b.png)

If someone want enable it then he can added it manually to this list